seacrowd/utils/constants.py

```diff
@@ -129,6 +129,7 @@ class Tasks(Enum):
 
     # No seacrowd schema
     FACT_CHECKING = "FCT"
+    NON_WORD_ERROR_SPELLING_CORRECTION = "NWESC"
```

Comment:
About the addition in `constants.py`, update with the latest master too, there seem to be merge conflicts.

Reply:
done